.envrc

```diff
@@ -0,0 +1 @@
+use flake .#controller
```

Comment:
I usually find that folks do commit their `.envrc` files. If folks are not using `direnv`, it will not affect them. If they do use `direnv`, but do not have Nix installed, nothing happens until they do `direnv allow` which would then try to enter the shell and fail. Then can then do `direnv deny` to disable direnv again. So overall, it's pretty harmless. Also, if folks are using VSCode, then it could help them get a working setup.